### PR TITLE
fix frontend failing tests and add missing js script

### PR DESCRIPTION
**Pull request by ricardofreitasrocha**
### JIRA link (if applicable) ###
[JM-6594](https://centralgovernmentcgi.atlassian.net/browse/JM-6594)

### Change description ###
- Fix a couple of failing tests
- Add missing gds script

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: client/templates/includes/scripts.njk [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <script src="{{ assetPath }}js/bundle.js"></script>
 
 <!--  Init GDS and MOJ toolkits -->
+<script src="{{ assetPath }}js/govuk/all.js"></script>
```
Comment (gregs-cgi) [inline client/templates/includes/scripts.njk:7]: Is this an unrelated change? Probably not actually a concern, but theoretically should be separate?

**Review by gregs-cgi — APPROVED** [review]

File: client/templates/includes/scripts.njk (hunk 1 above)
Comment (ricardofreitasrocha) [reply client/templates/includes/scripts.njk:7]: I missed this when I updated the gds toolkit yesterday :D removed by mistake but added as part of this fix
